pkg/vfs/writer.go

```diff
@@ -404,7 +404,11 @@ func (f *fileWriter) Flush(ctx meta.Context) syscall.Errno {
 
 func (f *fileWriter) Close(ctx meta.Context) syscall.Errno {
 	defer f.w.free(f)
-	return f.Flush(ctx)
+	eno := f.Flush(ctx)
+	if eno == 0 {
+		fileSizeHistogram.Observe(float64(f.GetLength()))
```

Comment:
The size of written file will be difficult to understand, it also increase the number of metrics a lot (41), I'd like to not have these.

Reply:
* For the bucket number - it's OK to cut it to a smaller number (eg. 24).
* For the metric - it tells us size distribution of user's files, allowing us to understand whether we are dealing with large or small files. Or any other suggestions to get this distribution?
<img width="453" alt="image" src="https://github.com/juicedata/juicefs/assets/2657334/ac7facec-ffa8-4337-bd9e-8080ebbd5497">

Reply:
you can know the avg size by total usage and inodes. For a given time period, you can know added files and added space. You can also find out the written data and write ops of metadata to find out the average size of chunk.

Reply:
OK, got it, will close.